src/main/Java/AnagramKata.java

```diff
@@ -0,0 +1,50 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class AnagramKata {
+    public static Scanner s;
```

Comment:
Interesting, why have you made the Scanner static?  Also `s` isn't the most descriptive name you could have used ;-)